### DISCUSSION TITLE
Significant Cpu Spikes on Daw Host Studio One 5.4 Mac M1 Win 10 Server

### DISCUSSION THREAD

**Discussion by Dimentagon** [apohl79/audiogridder#750] [Help]
I have noticed significant CPU Spikes on the Mac Mini M1 (OSZ 11.5) in Presonus Studio One 5.4 running in Native mode. The Host is a Windows 10 i7 latest generation Intel NUC.  I have tested the latest Beta in both machines and the problem persists so I have installed 1.1 however, the spikes are now more prevalent

**Comment by apohl79** [comment]
"CPU spikes" in the DAW performance meter usually mean latency spikes, as AG plugins do not use much CPU, they are more waiting for audio data to come back. Try to increase the buffer size.
